src/main/antlr/Asciidoc.g4

```diff
@@ -48,7 +49,7 @@ word            : CHARS PUNCTUATION? ;
 CHARS           : [0-9a-zA-Z]+ ;
 IDENTIFIER      : [a-zA-Z]+ ;
 //BEGIN_PUNCTUATION : [¿¡] ; // Some languages have beginning punctuation (e.g. Spanish)
-PUNCTUATION     : [.:,;] ;
+PUNCTUATION     : '.' ;//[.:,;] ;
 //word            : ~[\n\t ]+ ;
-ANYCHAR         : ~[\n\t ]+ ; // PH
+ANY_CHAR         : ~[\n\t ] ; // PH
```

Comment:
This is parsing most of the doc for me - The only thing that is failing now is a punctuation following a word without a space. I'm looking at some Antlr docs to see how we can handle this.